app/src/main/java/ru/kpfu/itis/fittrack/viewpager/ReceivingInformationFragment.kt

```diff
@@ -0,0 +1,120 @@
+package ru.kpfu.itis.fittrack.viewpager
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.EditText
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.widget.ViewPager2
+import ru.kpfu.itis.fittrack.R
+import ru.kpfu.itis.fittrack.databinding.FragmentReceivingInformationBinding
+
+class ReceivingInformationFragment : Fragment() {
+    private var _binding: FragmentReceivingInformationBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentReceivingInformationBinding.inflate(inflater, container, false)
+
+        var gender: Boolean? = null
+        with(binding) {
+            materialButtonToggleGroup.addOnButtonCheckedListener { _, checkedId, isChecked ->
+                if (isChecked) {
+                    when (checkedId) {
+                        R.id.btnFemale -> gender = true
+                        R.id.btnMale -> gender = false
+                    }
+                }
+            }
+            btnSaveUser.setOnClickListener {
+                if (!isEmpty(etAge, etHeight, etWeight) && isCorrect(
+                        etWeight,
+                        etHeight,
+                        etAge
+                    ) && gender !== null
```

Comment:
if (!isempty(...) && isCorrect(...)) {
    gender?.let {
        saveUserData(it, ...)